shell/browser/api/electron_api_utility_process.cc

```diff
@@ -265,28 +284,46 @@ void UtilityProcessWrapper::PostMessage(gin::Arguments* args) {
     return;
 
   blink::TransferableMessage transferable_message;
+
+  auto* isolate = args->isolate();
+  gin_helper::ErrorThrower thrower(isolate);
+
+  // |message| is any value that can be serialized to StructuredClone.
   v8::Local<v8::Value> message_value;
-  if (args->GetNext(&message_value)) {
-    if (!electron::SerializeV8Value(args->isolate(), message_value,
-                                    &transferable_message)) {
-      // SerializeV8Value sets an exception.
-      return;
-    }
+  args->GetNext(&message_value);
+
+  if (!electron::SerializeV8Value(isolate, message_value,
```

Comment:
It's not clear to me that this will fail gracefully if `message_value` is an empty handle. Since this is a behavior change, perhaps add a test?

Reply:
Whoops yes you're correct, it does not. Two potential solutions - we can default to undefined:

```
v8::Local<v8::Value> message_value = v8::Undefined(isolate);
```

or return to previous behavior. Do you have a preference?